recipes/xz/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "5.2.2" %}
+
+package:
+    name: xz
+    version: {{ version }}
+
+source:
+    fn:  xz-{{ version }}.tar.bz2
+    url: http://tukaani.org/xz/xz-{{ version }}.tar.gz
+    sha256: 73df4d5d34f0468bd57d09f2d8af363e95ed6cc3a4a86129d2f2c366259902a2
+
+build:
+    number: 0
+    skip: True  # [win]
```

Comment:
We just use 7zip on Windows at Continuum.  That said, it would be nice to build this also.  I have too much else going on right now, but you should tag the feedstock as needing Windows, and hopefully me or someone else can look at it soon.